src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import {FormsModule} from '@angular/forms';
 import {HttpClientModule} from '@angular/common/http';
 import {HomeComponent} from './home/home.component';
 import { FooterComponent } from './footer/footer.component';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
```

Comment:
@Panchoxa  doesn't it show you an error?
<img width="656" alt="Screenshot 2020-03-09 at 09 43 46" src="https://user-images.githubusercontent.com/28290808/76192441-7f0cac80-61ea-11ea-8deb-ffe5ca735e9e.png">

Reply:
![image](https://user-images.githubusercontent.com/53005412/76193023-cb0c2100-61eb-11ea-9289-f9a38f6093cd.png)
@kovalshmidt i have no any errors

Reply:
@Panchoxa 
<img width="387" alt="Screenshot 2020-03-09 at 10 04 08" src="https://user-images.githubusercontent.com/28290808/76193750-5df98b00-61ed-11ea-87a0-989609986314.png">